sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -360,48 +361,28 @@
     }
 
     /**
-     * Attempts to get a ConfigurationSetting that matches the {@code key}.
+     * Attempts to get a ConfigurationSetting that matches the {@code key}, the {@code label} as optional, and the
+     * {@code acceptDateTime} as optional.
      *
      * <p><strong>Code Samples</strong></p>
      *
-     * <p>Retrieve the setting with the key "prodDBConnection".</p>
+     * <p>Retrieve the setting with the key "prodDBConnection" and a time that one minute before now at UTC-Zone</p>
      *
-     * {@codesnippet com.azure.data.appconfiguration.configurationasyncclient.getSetting#string}
+     * {@codesnippet com.azure.data.appconfiguration.configurationasyncclient.getSetting#string-string-OffsetDateTime}
      *
      * @param key The key of the setting to retrieve.
+     * @param label Optional, the label of the setting to retrieve.
+     * @param acceptDateTime Optional, to access a past state of the configuration setting.
```

Comment:
All of these parameters that are optional should not have their parameter description start by saying 'Optional'. It doesn't read well and nor does it speak to what the user should provide if they don't want to provide anything - null, empty string, etc? I would have this formatted to say "The label of the setting to retrieve, or optionally, null if a setting with label is desired" - or whatever the actual description text should say.